process.c

```diff
@@ -4223,7 +4233,10 @@ rb_mjit_fork(void)
 
     after_fork_ruby();
     disable_child_handler_fork_parent(&old);
-    if (pid == 0) rb_thread_atfork();
+    if (pid == 0) {
+        clear_pid_cache();
+        rb_thread_atfork();
```

Comment:
Instead of inserting `clear_pid_cache()` wherever `rb_thread_atfork` is called, why not call it in `rb_thread_atfork`?

Reply:
because `rb_thread_atfork` is in another file so it can't call it. But I add an `rb_process_at_fork()` that calls both.

Reply:
Done.